.mvn/maven.config

```diff
@@ -1 +1 @@
--T 0.5C
+-T 1
```

Comment:
?

Reply:
There is a regression that causes concurrent module build to fail with a license plugin error more often lately. To avoid that I don't build modules concurrently. I'll revert this, it wasn't intentional.

Reply:
https://github.com/hazelcast/hazelcast/pull/23754/commits/62615e20d2af8af2bba2e4a4d1f6a2b45e54e77f